### PR TITLE
About page improvements

### PR DESCRIPTION
**Pull request by niknetniko**
Another version of the about page changes:

- Include text and some changes from #2150
- Include Elixir logo #1646 

screencapture-dodona-localhost-3000-about-2020-11-03-17_26_52

Todo:
- [x] Let others review the text

### CODE REVIEW

**Review by charvp — APPROVED** [review]

File: app/views/pages/_description.en.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<p>
+  Dodona is a <strong>free</strong> online exercise platform for learning to program. It wants to teach students how to
```
Comment (bmesuere) [inline app/views/pages/_description.en.html.erb:2]:
```suggestion
  Dodona is a <strong>free</strong> online exercise platform for learning to code. It wants to teach students how to
```

File: app/views/pages/_description.en.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  program in the most meaningful and effective way possible, by using different techniques to apply the advantages of
+  personalized coaching in contexts where students usually only have access to traditional coaching (e.g. lectures or
+  exercise sessions) or even no coaching at all (e.g. independent learning or homework).
+</p>
+<p>
+  Dodona acts as an online co-teacher, designed to give every student access to high quality education. The focus is on
+  automatic corrections and giving <strong>meaningful feedback</strong> on the submitted solutions from the students.
+</p>
+<p>
+  Teachers can create their own course, composed of different series of exercises. They can use already
```
Comment (bmesuere) [inline app/views/pages/_description.en.html.erb:12]:
```suggestion
  Teachers can create their own course, composed of different series of exercises. They can use
```

File: app/views/pages/_description.en.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  existing courses and exercises,
+  but can also create their own exercises and learning materials.
+  Thanks to the built-in learning analytics and datavisualisations it also is easy to monitor the progress of students.
+  <a href=https://dodona-edu.github.io/en/guides/>Here</a> you'll find a number of guides, primarily focused on
```
Comment (bmesuere) [inline app/views/pages/_description.en.html.erb:16]:
```suggestion
  <a href=https://dodona-edu.github.io/en/guides/>Here</a>, you'll find a number of guides, primarily focused on
```

File: app/views/pages/_description.en.html.erb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  teachers.
+</p>
+<p>
+  The Dodona platform contains <strong>hundreds of programming exercises</strong> that can be used to master several
+  programming languages.
+  All exercises come with automatic feedback on correctness, execution time and/or programming style. You can sign in
+  with the credentials of your institution.
```
Comment (bmesuere) [inline app/views/pages/_description.en.html.erb:23]:
```suggestion
  with the credentials of your educational institution.
```

File: app/views/pages/_team.en.html.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<div class="col-xs-12 col-sm-4">
+  <%= image_tag("ugent.en.png", class: 'img-responsive') %>
+</div>
+<div class="col-xs-12 col-sm-8">
+  <h3>Developed at Ghent University</h3>
+  <p>
+    Dodona is developped by a team from the research group
+    <a href="https://www.ugent.be/we/twist/en/research/computational-biology.htm">Computational Biology</a> from the
```
Comment (bmesuere) [inline app/views/pages/_team.en.html.erb:8]:
```suggestion
    Dodona is developed by a team from the 
    <a href="https://www.ugent.be/we/twist/en/research/computational-biology.htm">Computational Biology</a> research group from the
```

**Review by bmesuere — APPROVED** [review]
👍 a few minor textual changes

**Pull request merged**